nautobot/docs/installation/nautobot.md

```diff
@@ -126,40 +134,44 @@ Before you can use Nautobot, you'll need to configure it by telling it where you
 
 ### Initialize your configuration
 
-Initialize a new configuration by running `nautobot-server init`. You may specify an alternate location and detailed instructions for this are covered in the documentation on [Nautobot Configuration](../../configuration).
+Initialize a new configuration by running `nautobot-server init`. You may specify an alternate location and detailed instructions for this are covered in the documentation on [Nautobot Configuration](/configuration).
```

Comment:
Are you sure that changing relative paths to absolute paths works correctly both in `mkdocs` and in readthedocs (especially - is this going to end up linking to `nautobot.readthedocs.io/configuration` instead of `nautobot.readthedocs.io/en/latest/configuration`)? Relative links "feel" more correct to me in any case, so it's not clear to me why this needed to be changed.

Reply:
Yes. I've been using `mkdocs serve` to build and review the docs in real time as I edit them.

I feel like this needed to be changed because it's actually easier to get right, when you know which page you're linking to vs. having to remember where in the tree you are and how many levels to go back. It also makes links more portable if we decide to move things around.